book/tests/snapshots/snapshot__run_display_reference_prql@ranges-1.prql.snap

```diff
@@ -0,0 +1,14 @@
+---
+source: book/tests/snapshot.rs
+expression: "Item::Query(parse(&prql).unwrap())"
+input_file: book/tests/prql/language-features/ranges-1.prql
+---
+prql dialect: generic
+
+from orders
+sort [
+  !value,
```

Comment:
This looks like a bug! Nice tests!

Reply:
(this doesn't need to be fixed to merge tbc)

Reply:
oops sorry, this snapshot was not from the latest commit with display trait fixes